apps/snitch_api/lib/snitch_api/address_context.ex

```diff
@@ -0,0 +1,87 @@
+defmodule SnitchApi.AddressContext do
```

Comment:
This should actually be a `Checkout` context, what say. It should not be specific to address only. It can also have functions like `Checkout.payment_methods`.

Reply:
phoenix structure says to have a context folder where. Like `snitch_api/checkout/checkout.ex` `SnitchApi.Checkout` @kanishkablack